src/test/fixtures/deepJsonDefinitions/CaseEvent/Initiation/initiateCase.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "DRAFT",
+    "ID": "initiateCase",
+    "Name": "Initiate Case",
+    "Description": "Create a new case",
+    "DisplayOrder": 1,
+    "PostConditionState": "1_Initiation",
+    "SecurityClassification": "Public",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_BASE_URL}/initiate/callback"
+  },
+  {
```

Comment:
I know it is just sample file but for be it is counter intuitive to have submit event in JSON file named `Initiation/initiateCase.json`. Can you remove it? I do not think it adds much to test scenario.

Reply:
Sure, I think it was just a copy and paste anyway

Reply:
Or you could have:

`Application/initiateCase.json`
`Application/submitCase.json`

and 

`Case/revertCase.json`

What I am trying to achieve is ot have sample date that reflects problem that we try to solve. Having clear example will allow us saying to others "Hey we added that cool feature to processor so address the following problem, here is example of the problem that we addressed"

Reply:
Yeah, makes sense